### DISCUSSION TITLE
Use FSR 2.X with GTA-V.

### DISCUSSION THREAD

**Discussion by blankuserrr** [NarutoUA/gta5_fsr#85] [Ideas]
Yes, I know that GTA-V doesn't **_directly_** support DX12, but you can incorporate [this](https://github.com/doitsujin/elf-auf-zwoelf) into it. It translates D3D11 games to D3D12 using Microsoft's own [D3D11On12](https://docs.microsoft.com/en-us/windows/desktop/direct3d12/direct3d-11-on-12) wrapper library.

**Comment by NarutoUA** [comment]
Whats the point if this wrapper reduces performance anyway?

> If everything works correctly, you will most likely experience reduced performance compared to native D3D11, and in some cases rendering issues and/or driver crashes. Some games may refuse to run entirely.

**Reply by blankuserrr** [reply]
i mean yeah but still
